content/en/contribute/_index.md

```diff
@@ -20,4 +20,17 @@ The [Development](https://forum.communityhealthtoolkit.org/c/developement/7) sec
 - [Share your expertise](https://forum.communityhealthtoolkit.org) on our Forum
 - Sign up for our [newsletter](https://communityhealthtoolkit.org/contact) to get the latest updates
 
+**Are you a partner wondering how issues are prioritized?**
+
+Issues that a partner will: 
```

Comment:
Referred to as "tickets" below. I tend to prefer this since "issues" is GitHub specific and otherwise sounds like something is a problem. Not that big of a deal though, as long as we're consistent.

```suggestion
Tickets that a partner will: 
```